### PR TITLE
Add artifact parameter to provide image download

### PR DESCRIPTION
**Pull request by roesekoSICKAG**
Proposal to fix issue #95

### CODE REVIEW

**Review by mrentsch65 — APPROVED** [review]

**Review by OlafWestrik — APPROVED** [review]

File: JSON_for_IO-Link.yaml [hunk 1]
```diff
@@ -1080,12 +1080,13 @@ paths:
       operationId: GetIoddsFile
       tags:
         - iodds
-      summary: Get a specific IODD. Optional.
-      description: All query parameters are mandatory.
+      summary: Get a specific IODD or other IODD artifacts like device image. Optional.
```
Comment (julian-st) [inline JSON_for_IO-Link.yaml:1084]: What is optional?

File: JSON_for_IO-Link.yaml [hunk 2]
```diff
@@ -1094,6 +1095,10 @@ paths:
...
...I3cC1pt+bNXXUrVPuPUNI2+fe9j4aIEXhr7MsXFJXy7fQdxM2Yya/ZM/Pz8bLee6PV6XnxxLC/9baxlKHa0FaWdOcvU2KlkZ2dTUFBA/Pz57N93ADChlKJrt9tZuCjRsmfueiY41vHqtbZ0nU5H37738dHHHzFpymsEVq7fOoCrqytP/fkJkpI+ITIyssmFrN7wqr8DZX3c/lbBG7RSpBxLPWmaxoMPPsjceW/i5KQDzO9bLVjwFgUFBfb/QkZJSQlrP1nLm7Pnkp+f36DWNuyJR2kd1JrFi5ZUVKSOwX94iGlvTLuuF+CLior49tvvKnZDN2zOqnfT06/f/bUOr0opsrOzOXz4CPv37ePkzz+TkZ5hWbXQ610JCgqibdu29OzVk27d7qBd23a4uLo0qYjZ2dns27cfVMO16tQp3LID5+LFi+zduw+UwsXVlfvvj0Cv16OUYu/efVy8eAmAtm1DLBu8a6OsrIxt276ivNy8Wtez5z34+5s34qSknOD06dMopfD19aFP7z5oOq2O3r+UXbt2U1hQaFndCgsLq/3PtpjKTRw+coSlS5awZfO2emOI6pMNML8VOH7CK/zpT4/g7e3Nb4Hy8nLLy1KVK1ROTk64u7vj4uIifxfpRmRBVT1JSYPBwMGDB/l0/QY2btyEobCoRtdtvdSn0zkR3vE2nn32GfoPGEBgYMA1/T0ZQaSsF6PRSFZWFr/8kkpa2hmOHTtunsRo4OHRjN916UK70FA6dOhAaGg73N3db8gfgRL+i6UUhBuFjKuCSCkIIqUgUgqCSCmIlIIgUgoipSCIlIIgUgoipSCIlIJIKQgipSBSCoJIKYiUgiBSCoJIKYiUgiBSCiKlIIiUgkgpCCKlIIiUgkgpCCKlIFIKgkgpiJSCIFIKIqUgiJSCIFIKIqUgiJSCSCkIIqUgUgqCSCkIIqUgUgqCSCmIlIIgUgoipSCIlIJIKQgipSCIlIJIKQgipSBSCoJIKYiUgiBSCoJIKYiUgiBSCiKlIIiUgkgpCCKlIFIKgkgpCLXxvz6Kf20HDytIAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIxLTEwLTA2VDExOjEzOjAxKzAyOjAwSxzPnwAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMS0xMC0wNlQxMToxMzowMiswMjowMAupbb4AAAAASUVORK5CYII="
```
Comment (julian-st) [inline JSON_for_IO-Link.yaml]: This example is too long for my taste and different to the application/zip answer

File: JSON_for_IO-Link.yaml (hunk 1 above)
Comment (roesekoSICKAG) [reply JSON_for_IO-Link.yaml:1084]: The optional means that this endpoint can be implemented and is not mandatory.

File: JSON_for_IO-Link.yaml (hunk 2 above)
Comment (roesekoSICKAG) [reply JSON_for_IO-Link.yaml]: Good point. I picked a different picture as an example now.

**Comment by roesekoSICKAG** [comment]
@julian-st, @OlafWestrik: Could you please review this PR?

**Review by lorandmolnar — APPROVED** [review]

**Pull request merged**